canvasapi/canvas.py

```diff
@@ -230,7 +250,9 @@ def create_appointment_group(self, appointment_group, **kwargs):
 
         return AppointmentGroup(self.__requester, response.json())
 
-    def create_calendar_event(self, calendar_event, **kwargs):
+    def create_calendar_event(
+        self, calendar_event: dict, **kwargs: Union[str, dict, Optional[dict]]
```

Comment:
Matt and I have had a few conversations about this and we're not sure what the best approach is: how do we handle `kwargs`?

One of the nice things about taking in arbitrary keyword arguments is that canvasapi can maintain parity with Canvas updates as long as the endpoints themselves don't change. The parameters might change types or names or expect new values, but there's nothing we have to do downsteam.

Of course, the downside to that is that there's no useful information for developers in the function signature about what Canvas might accept. Typehints partially address this, but they reintroduce the API parity problem. 

Unless we can come up with a good tradeoff, I wonder if our `kwargs` typehint needs to be `Any` or something equally permissive.

Reply:
Yeah, I did a lot of reading on how to type `**kwargs` and there isn't really a broad consensus. For me, I'm typically passing those in as a `dict`, but `Any` also makes sense. It's not considered bad practice because it really could be anything. Maybe a note in the docs somewhere about the `Any` type being limited to `kwargs` specifically?